source/translators/how-tos/how-to-join-a-translation-team.rst

```diff
@@ -29,6 +29,6 @@ To join a translation team you must complete these steps:
 .. _Transifex: https://www.transifex.com/signup/
 .. _getting   started guide: https://docs.transifex.com/getting-started-1/translators
 .. _Translating with the Web Editor: https://docs.transifex.com/translation/translating-with-the-web-editor
-.. _edx-platform project: https://explore.transifex.com/open-edx/edx-platform/
+.. _edx-platform project: https://explore.transifex.com/open-edx/openedx-translations/
```

Comment:
I think we should change the text here too, as it still says "Join the Transifex `edx-platform project`_ "